evap/staff/forms.py

```diff
@@ -333,6 +336,8 @@ def __init__(self, *args, **kwargs):
         self.fields['courses_participating_in'].initial = courses_of_current_semester.filter(participants=self.instance) if self.instance.pk else ()
         self.fields['courses_participating_in'].label = _("Courses participating in (active semester)")
         self.fields['courses_participating_in'].required = False
+        self.fields['is_staff'].initial = self.instance.is_staff
+        self.fields['is_grade_user'].initial = self.instance.is_grade_publisher
```

Comment:
this broke the tests, it's probably enough to put these two lines into a `if self.instance.pk`. the problem is that you can't access m2m relationships (like groups on a user) before it has been saved to the database once because you need the user's id for that. this happens e.g. when using this form for creating a user.

i saw some other stuff i'd like to change in this method so i'll fix the tests while i'm at it. thanks again for the PR!